src/pocketmine/item/Item.php

```diff
@@ -684,6 +684,21 @@ public function getLore() : array{
 		return [];
 	}
 
+	public function addLore(string $lore) {
+		$tag = $this->getNamedTag() ?? new CompoundTag("", []);
+		if(!isset($tag->display)){
+			$tag->display = new CompoundTag("display", []);
+		}
+		
+		if(!isset($tag->display->Lore)) {
+			$tag->display->Lore = new ListTag("Lore");
+		}
+		
+		$tag->display->Lore->setTagType(NBT::TAG_String);
+		$tag->display->Lore[$tag->display->Lore->getCount()] = new StringTag("", $lore);
```

Comment:
Even though both work you're using two different methods in one line now. I'd say change it to what @Muqsit recommends.